.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -98,6 +98,29 @@ The integration reached or maintains the following [Integration Quality Scale][q
 - [ ] 🥇 Gold
 - [ ] 🏆 Platinum
 
+<!--
+  This project is very active and we have a high turnover of pull requests.
+
+  Unfortunately, the number of incoming pull requests is higher than what our
+  reviewers can review and merge so there is a long backlog of pull requests
+  waiting for review. You can help here!
+  
+  By reviewing another pull request, you will help raise the code quality of
+  that pull request and the final review will be faster. This way the general
+  pace of pull request reviews will go up and your wait time will go down.
+  
+  When picking a pull request to review, try to choose one that hasn't yet
+  been reviewed.
+
+  Thanks for helping out!
+-->
+
+To help with the load of incoming pull requests:
+
+- [ ] I have reviewed two other [open pull requests][prs] in this repository.
+
+[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-asc+-review%3Aapproved
```

Comment:
Codacy found an issue: [[final-definition] Move definitions to the end of the file (after the node at line `124`)](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)